packages/artifact/package.json

```diff
@@ -36,10 +36,10 @@
     "url": "https://github.com/actions/toolkit/issues"
   },
   "dependencies": {
-    "@actions/core": "^1.2.1",
-    "@actions/http-client": "^1.0.7",
-    "@types/tmp": "^0.1.0",
-    "tmp": "^0.1.0",
+    "@actions/core": "^1.2.4",
+    "@actions/http-client": "^1.0.8",
+    "@types/tmp": "^0.2.0",
+    "tmp": "^0.2.1",
```

Comment:
What's the benefit of doing this?

If there's a security issue with any of these, the alert should be enough of a notice. It feels like we're just introducing extra code changes proactively without needing the updated packages.

Reply:
It's recommended to regularly update them. From: https://docs.npmjs.com/updating-packages-downloaded-from-the-registry

> We recommend regularly updating the local packages your project depends on to improve your code as improvements to its dependencies are made.

Possibly fewer security alerts is just one benefit.  Updating these packages doesn't break compat since there in ways that packages define the min and max versions that they can use, so by doing this we are regularly pulling in improvements. 

---

Aside as to why i prefer [ncu over npm update](https://stackoverflow.com/questions/57069622/command-npm-update-vs-package-npm-check-updates)

Reply:
Reverted after some offline discussions, only thing left in this PR is the clarifications around headers.